### ISSUE TITLE
Adding timeout to ChatOpenAI raises ConfigError

### ISSUE THREAD

**Issue by andreas-thomik** [langchain-ai/langchain#13124]
### System Info

Running langchain==0.0.332 with python 3.11 and openai==1.2.0 on Windows.

### Information

- [ ] The official example notebooks/scripts
- [ ] My own modified scripts

### Related Components

- [X] LLMs/Chat Models
- [ ] Embedding Models
- [ ] Prompts / Prompt Templates / Prompt Selectors
- [ ] Output Parsers
- [ ] Document Loaders
- [ ] Vector Stores / Retrievers
- [ ] Memory
- [ ] Agents / Agent Executors
- [ ] Tools / Toolkits
- [ ] Chains
- [ ] Callbacks/Tracing
- [ ] Async

### Reproduction

```python
from langchain.chat_models import ChatOpenAI

ChatOpenAI(
    model="gpt-3.5-turbo",
    request_timeout=5,
)
```

### Expected behavior

Should run without errors.

Likely due to newly introduced `httpx.Timeout` type in `request_timeout` (https://github.com/langchain-ai/langchain/pull/12948). Always importing httpx and tiktoken (i.e. not conditionally on `TYPE_CHECKING`) fixes the issue.

**Comment by baskaryan** [comment]
Thanks for flagging @andreas-thomik. #13154 should fix and will be released in 0.0.334, let me know if you still see it after that

**Comment by andreas-thomik** [comment]
Seems fixed to me, thank you!
